src/app/components/MostRead/Canonical/List/index.test.tsx

```diff
@@ -110,4 +110,18 @@ describe('MostReadList', () => {
     );
     expect(container).toMatchSnapshot();
   });
+  it('should render burmese most read with a one column layout', () => {
```

Comment:
I am wondering if instead of doing (yet another 😿) snapshot test here, is there a way in which we could check that the correct column layout is set? Is there an attribute / css property we can check instead? (and we could maybe have an opposite test to check that it's 5 when the service is not burmese?)

Reply:
Can we do this as part of the tech work to replace snapshot tests with more specific ones? We talked about it in the tech prioritisation last week as something to do over a lot of the tests.

We could close off this PR and deal with improving the test as a separate task?